src/interface/src/styleguide/scenario-card/scenario-card.component.scss

```diff
@@ -0,0 +1,101 @@
+@import 'mixins';
+@import 'colors';
+
+:host {
+  @include xs-label();
+  box-sizing: border-box;
+  height: 140px;
+  left: 0px;
+  right: 0px;
+  top: 40px;
+  background: #ffffff;
+  border: 1px solid $color-soft-gray;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+  display: inline-flex;
+  padding: 8px 12px;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+  flex-direction: column;
+  gap: 1em;
+
+  &.disabled-content {
+      color: $color-text-gray;
+  }
+
+  .title-row {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 3em;
+    justify-content: space-between;
+  }
+  .title-section {
+    display: flex;
+    flex-direction: row;
+    gap: 1em;
+    align-items: center;
+  }
+  .controls-section {
+    display: flex;
+    flex-direction: row;
+    gap: 0;
+  }
+
+  h4 {
+    @include h4();
+  }
+
+  .title-chip {
+    height: 24px;
```

Comment:
Not sure where we landed on this detail, do we remove this or do a variant? On figma its 32px so I don't think we need this, unless Im looking at the wrong file :D